cairo/scripts/simple_math/src/simple_math.cairo

```diff
@@ -0,0 +1,41 @@
+fn sum(number1: u128, number2: u128) -> u128 {
+    number1 + number2
+}
+
+fn subtract(number1: u128, number2: u128) -> u128 {
+    number1 - number2
+}
+
+fn multiply(number1: u128, number2: u128) -> u128 {
+    number1 * number2
+}
+
+fn divide(number1: u128, number2: u128) -> u128 {
+    assert(number2 != 0, 'Division by zero is not allowed');
+    number1 / number2
+}
+
+fn modulus(number1: u128, number2: u128) -> u128 {
+    assert(number2 != 0, 'Modulus by zero is not allowed');
+    number1 % number2
+}
+
+fn main() {
+    let number1: u128 = 22;
+    let number2: u128 = 7;
+
+    // Sum example
```

Comment:
Can you print those comments before every operation? For example In this case print `-> Sum Example:` and the same with the others?